modules/data-storage-management.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * storage/optimizing-storage.adoc
+
+[id="data-storage-management_{context}"]
+= Data storage management
+
+The following table summarizes the main directories that {product-title} components write data to.
+
+.Main directories for storing {product-title} data
+[options="header,footer"]
+|===
+|Directory|Notes|Sizing|Expected growth
+
+|*_/var/lib/openshift_*
+|Used for etcd storage only when in single master mode and etcd is embedded in the *atomic-openshift-master* process.
+|Less than 10 GB.
+|Will grow slowly with the environment. Only storing metadata.
+
+|*_/var/lib/etcd_*
+|Used for etcd storage when in Multi-Master mode or when etcd is made standalone by an administrator.
+|Less than 20 GB.
+|Will grow slowly with the environment. Only storing metadata.
+
+|*_/var/lib/docker_*
+|When the runtime is docker, this is the mount point. Storage used for active container runtimes (including Pods) and storage of local images (not used for registry storage). Mount point should be managed by docker-storage rather than manually.
+|50 GB for a Node with 16 GB memory.
+
+Additional 20-25 GB for every additional 8 GB of memory.
+|Growth is limited by the capacity for running containers.
+
+|*_/var/lib/containers_*
+|When the runtime is CRI-O, this is the mount point. Storage used for active container runtimes (including Pods) and storage of local images (not used for registry storage).
+|50 GB for a Node with 16 GB memory.
+
+Additional 20-25 GB for every additional 8 GB of memory.
+|Growth limited by capacity for running containers
+
+|*_/var/lib/origin/openshift.local.volumes_*
```

Comment:
Looks in OCP4.x, this directory should be `/var/lib/kubelet`, @sunilcio could you help double confirm?

Reply:
@qinpingli yeah correct, I see it should be `/var/lib/kubelet`

Reply:
@qinpingli or @sunilcio - I changed this to `*_/var/lib/kubelet/openshift.local.volumes_*` but want to confirm, is this correct? Or should it just be `*_/var/lib/kubelet` (without `openshift.local.volumes_*`)?